lib/pltr/v2/actions/lines.js

```diff
@@ -42,8 +50,58 @@ export function editLineColor(id, color) {
   return { type: EDIT_LINE_COLOR, id, color }
 }
 
-export function reorderLines(lines, bookId) {
-  return { type: REORDER_LINES, lines, bookId }
+export const reorderLines =
+  (droppedPosition, originalPosition, lines, bookId) => (dispatch, getState) => {
```

Comment:
Can we do the same thing as the other action here: remove `lines` and `bookId` and select those from the state.
As a general rule, we should only take a minimal amount of data as arguments.

Reply:
@Quiescent sorry i didnt double check. Thank you for checking

Reply:
@jeanawhou, we might be missing each other here.  I'm suggesting that we change the API of this function:
```
export const reorderLines =
  (droppedPosition, originalPosition, lines, bookId) => (dispatch, getState) =>
```
->
```
export const reorderLines =
  (droppedPosition, originalPosition) => (dispatch, getState) =>
```

Reply:
@Quiescent sorry i forgot about the reorderlines reducer 🥲

Reply:
Just updated the PR. Thank you for checking 🙇